geomstats/_backend/tensorflow/linalg.py

```diff
@@ -9,11 +9,16 @@
 eigvalsh = tf.linalg.eigvalsh
 expm = tf.linalg.expm
 inv = tf.linalg.inv
-norm = tf.linalg.norm
 sqrtm = tf.linalg.sqrtm
 diagonal = tf.linalg.diag_part
 
 
+def norm(x, **kwargs):
+    if x.dtype != tf.float32 or x.dtype != tf.float64:
+        x = tf.cast(x, tf.float32)
```

Comment:
maybe allow the user to explicitely pass dtype? If working in float64 elsewhere this will break

Reply:
Yes, but the tf.linalg.norm does not work with tf.int32:
```
a  = tf.constant([1, 2, 3])
tf.linalg.norm(a)
```
fails, so we should remember that putting tf.int32 as dtype will fail.

Reply:
Done.